From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Fri, 5 Jun 2026 17:15:22 +0800
Subject: local-fhs-linux-paths

# DP: Description: Correct linux paths for FHS
# DP: Author: Unknown
# DP: Upstream status: Debian-Specific
# DP: Status Details: GNU doesn't follow the FHS.
# DP: Date: Unknown

# DP: Description: Correct linux paths for FHS
# DP: Author: Unknown
# DP: Upstream status: Debian-Specific
# DP: Status Details: GNU doesn't follow the FHS.
# DP: Date: Unknown
---
 sysdeps/generic/paths.h         | 2 +-
 sysdeps/unix/sysv/linux/paths.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sysdeps/generic/paths.h b/sysdeps/generic/paths.h
index 6b8f440..4e641c9 100644
--- a/sysdeps/generic/paths.h
+++ b/sysdeps/generic/paths.h
@@ -68,7 +68,7 @@
 /* Provide trailing slash, since mostly used for building pathnames. */
 #define	_PATH_DEV	"/dev/"
 #define	_PATH_TMP	"/tmp/"
-#define	_PATH_VARDB	"/var/db/"
+#define	_PATH_VARDB	"/var/lib/misc/"
 #define	_PATH_VARRUN	"/var/run/"
 #define	_PATH_VARTMP	"/var/tmp/"
 
diff --git a/sysdeps/unix/sysv/linux/paths.h b/sysdeps/unix/sysv/linux/paths.h
index 1342ab3..c764e20 100644
--- a/sysdeps/unix/sysv/linux/paths.h
+++ b/sysdeps/unix/sysv/linux/paths.h
@@ -68,7 +68,7 @@
 /* Provide trailing slash, since mostly used for building pathnames. */
 #define	_PATH_DEV	"/dev/"
 #define	_PATH_TMP	"/tmp/"
-#define	_PATH_VARDB	"/var/db/"
+#define	_PATH_VARDB	"/var/lib/misc/"
 #define	_PATH_VARRUN	"/var/run/"
 #define	_PATH_VARTMP	"/var/tmp/"
 
